frontend/src/components/Documentation.vue

```diff
@@ -319,14 +320,12 @@
               <p>
                 The
                 <i>Data overlay</i>
-                sidebar allows uploading of the custom data in
+                sidebar also allows uploading of custom data in
                 <a target="_blank" href="https://en.wikipedia.org/wiki/Tab-separated_values">
                   TSV format</a
-                >. If the file is parsed correctly, the file name will be highlighted in green; in
-                case errors are detected, it will be highlighted in red. The expected custom data
-                file should contain at least two columns with headers and using tab delimiter. The
-                first column has to contain gene IDs, identical to the ones in the model. The header
-                of the first column should be named as
+                >. The custom data file should contain at least two columns with headers and use tab
+                delimiter. The first column has to contain gene IDs, identical to the ones in the
```

Comment:
The first column can be different types of IDs:
```suggestion
                delimiter. The first column has to contain model IDs for genes, reactions or metabolites.
```